kong/plugins/datadog/schema.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  fields = {
+    host = {type = "string", default = "localhost"},
```

Comment:
This should be `required` and probably should not have a `default`, since datadog does not usually runs locally right?